src/components/create-contingency-list-form.js

```diff
@@ -150,40 +150,47 @@ export const CreateContingencyListForm = ({ open, onClose }) => {
     };
 
     const handleCreateNewContingencyList = () => {
-        if (contingencyListName === '') {
-            setCreateContingencyListErr(
-                intl.formatMessage({ id: 'elementNameErrorMsg' })
-            );
-            return;
-        }
+        //We don't do anything if the checks are not over
+        if (
```

Comment:
You could replace this by :    
if(contingencyListName === '' ||
            loadingCheckContingencyName ||
            !contingencyNameValid
){
// no need to manage CreateContingencyListErr here because it's managed in handlechange
return;
}
to avoid indenting all the code bellow